gradio/themes/__init__.py

```diff
@@ -14,3 +21,65 @@ def builder(*args, **kwargs):
     from gradio.themes.builder import demo
 
     return demo.launch(*args, **kwargs)
+
+
+_built_in_themes: Dict[str, ThemeClass] = {
+    t.name: t
+    for t in [
+        Base(),
+        Default(),
+        Monochrome(),
+        Soft(),
+        Glass(),
+    ]
+}
+
+# You can set the default theme to one of the built-in themes by setting the environment variable GRADIO_DEFAULT_THEME.
+# If the environment variable is not set or does not refer to a built-in theme, the default theme is Default.
+Context.default_theme_class = _built_in_themes.get(
```

Comment:
I think this would be better if it was the theme object rather than the class. That way, you could load a theme from the Hub and assign it as a global theme as well

Reply:
In which case, I'd rename `Context.default_theme_class` to `Context.default_theme`